### DISCUSSION TITLE
文章校正ツールが欲しい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#8] [要望・アイデア・不具合]
Web 上で [textlint](https://github.com/textlint/textlint) などの文章校正ツールが実行され、指摘してほしいです。
他にも [RedPen](https://redpen.cc/docs/latest/index_ja.html) や [prh](https://github.com/prh/prh) があると嬉しいです。

**Comment by kskwtnk** [comment]
@uhooi 

ご意見ありがとうございます！

そうですよね、私も普段記事を書いていますが、誤字のあるまま投稿してしまった経験あるのでとても分かります……。

現在力を入れていることの1つにエディタの改修があり、リリースできる時期は未定なのですがそのプロジェクトの一環として検討させていただきます！

**Reply by uhooi** [reply]
typo は自動で直してほしいですよねｗ
ローカルで検証できるので必須ではありませんが、ご検討いただけると嬉しいです…！

**Reply by Nakatai-0322** [reply]
確かに，日本語的におかしな文章がそのまま投稿されてしまうのはほんとに嫌ですよね。

**Reply by munierujp** [reply]
markdownlintあたりも走ってくれると嬉しいですね

**Reply by Nakatai-0322** [reply]
> markdownlintあたりも走ってくれると嬉しいですね

確かに！
名案ですね。

**Reply by kskwtnk** [reply]
活発なDiscussionありがとうございます！

@Nakatai-0322 @munierujp 

ちなみにイメージ的にはlinterとformatterでいうとどちらでしょう？
また、例えばformatterにしても自動でかかるものと明示的に走らせるものと……多分いくらか種類はあるのかなと思いまして、そういったイメージが共有いただけると非常にありがたいです！

**Comment by kimkim0814** [comment]
@uhooi
お待たせいたしました!

文章修正提案機能をベータ版として公開しました！
記事や質問を執筆する際に、文章に誤りや修正提案がある場合はエディタ上で提案を行われるようになりました。

この度はご意見をいただきありがとうございました。

https://qiita.com/release-notes/2023-10-30

**Reply by nak435** [reply]
ありがとうございます。
エディットモードの時の文字をもう少し大きくできないでしょうか？
個人的にはプレビューに近い大きさがいいのですが。Mac & Safari です。


<img width="300" alt="scr1" src="https://github.com/increments/qiita-discussions/assets/42625437/3044ff70-9bc0-4b2c-b9c4-1e5fd2ad4c04" />

<img width="300" alt="scr2" src="https://github.com/increments/qiita-discussions/assets/42625437/92592b58-978e-4fd6-9968-93bf52d5f844" />

**Reply by tanaka-asami415** [reply]
@nak435 
ご意見ありがとうございます🙏
お日にちのお約束はできませんが、font-size:16pxに変更しようと思います！（コメントのプレビューと同じ文字の大きさです）

また、こちらのDiscussionとは内容が異なるようなので、もしかして別のDiscussion へのお返事だったでしょうか？🙏

**Reply by nak435** [reply]
ありがとうございます。

> こちらのDiscussionとは内容が異なるようなので、もしかして別のDiscussion へのお返事だったでしょうか？

「文章修正提案機能のベータ版」を適用してから文字サイズが小さくなったので、こちらのDiscussionへコメントしましたが、違いましたか？

**Reply by tanaka-asami415** [reply]
@nak435 
> 「文章修正提案機能のベータ版」を適用してから文字サイズが小さくなったので、

なるほど、そういうことでしたか！教えていただきありがとうございます🙏

特にフォントサイズは変えていないのですが、おそらく下記のリリース「コメントフォームのUIを改善」で印象が変わったせいかなと思われます。
https://qiita.com/release-notes/2023-10-31

改善予定ですので、お時間いただけますと幸いです🙇

**Reply by tanaka-asami415** [reply]
@nak435 
お待たせしました！
こちら修正いたしました！✨

**Comment by getty104** [comment]
ベータ版として公開していた文章修正提案機能ですが、この度正式版として公開しました！
改めて、ご意見をいただきありがとうございました！
[リリースノート](https://qiita.com/release-notes/2024-01-31#%E6%96%87%E7%AB%A0%E4%BF%AE%E6%AD%A3%E6%8F%90%E6%A1%88%E6%A9%9F%E8%83%BD%E3%82%92%E6%AD%A3%E5%BC%8F%E7%89%88%E3%81%A8%E3%81%97%E3%81%A6%E5%85%AC%E9%96%8B%E3%81%97%E3%81%BE%E3%81%97%E3%81%9F)
